README.md

```diff
@@ -10,8 +10,10 @@ modeling, reporting, and data integrity work.
   Documentation for all CCAO data lakehouse tables and views
 - [:nut_and_bolt: dbt README](/dbt/README.md) - How to develop CCAO data
   infrastructure using dbt
-- [:test_tube: dbt Test Documentation](/dbt/tests/generic/README.md) -
-  Definitions for CCAO generic dbt data tests
+- [:test_tube: dbt QC Tests and Reports](dbt/README.md#-how-to-add-and-run-qc-tests-and-reports) -
+  How to add and run QC tests and reports using dbt
```

Comment:
I figure that QC testing and reporting is an important enough feature of this repo that it's worth linking to directly from the root README.

Maybe this is a personal preference thing, but the 🧪 emoji makes me think the link will be discussing QC products (as opposed to dbt generics), so I moved it from the generic docs to the QC  products docs and switched the generic docs to the 📝 emoji.